### PR TITLE
setup: file structure and workflow

### PR DESCRIPTION
**Pull request by shashidwivedi**
Create folder structure and workflow for merging CSS files for the component library

Note: You may see hard-coded values for colors and spacing styles missing in documentation, which will be pushed as I develop components subsequently.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+                eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+                ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+                aliquip ex ea commodo consequat.
+              </p>
+            </code>
+          </pre>
+        </article>
+        <footer class="footer">
+          <p><span>© Foobar UI</span></p>
+        </footer>
```
Comment (ttahm3d) [inline index.html]: move `footer` outside main

**Review by ttahm3d — COMMENTED** [review]
Have added one reveiw please check

**Review by ttahm3d — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600;700;900&family=Poppins:wght@300;400;600;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/base16/dracula.min.css"
+    />
```
Comment (nishasen) [inline index.html:20]: You can create one styles.css file, and import all the stylesheets there and add that stylesheet link here, no need to import all of them together in one file

**Review by nishasen — COMMENTED** [review]
Your folder structure looks clean, have suggested few changes do look into the comment

File: index.html (hunk 1 above)
Comment (shashidwivedi) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (shashidwivedi) [reply index.html:20]: Hi Nisha,
As we are writing in plain CSS and also several imports can cause resource downloads, I am not using CSS imports in this case. If you see the 'workflow' folder, there is a bash script to merge all those CSS files. In that way, anyone can use the single CSS file in their link tag for using it.
PS: I will use @imports when I will move this CL from CSS to SASS as @imports in SASS behave differently and do not load resources.

**Review by nishasen — APPROVED** [review]

**Pull request merged**